### PR TITLE
Fix existing CheckStyle issues (GlowChunkGenerator--GlowEntity)

### PR DESCRIPTION
**Pull request by Pr0methean**

### CODE REVIEW

File: src/main/java/net/glowstone/chunk/GlowChunkSnapshot.java [hunk 1]
```diff
@@ -11,17 +12,31 @@
  */
 public class GlowChunkSnapshot implements ChunkSnapshot {
 
-    private final int x, z;
+    @Getter private final int x;
```
Comment (mastercoms) [inline src/main/java/net/glowstone/chunk/GlowChunkSnapshot.java]: Separate lines for annotations.

Comment (Pr0methean) [reply src/main/java/net/glowstone/chunk/GlowChunkSnapshot.java]: Done.

File: src/main/java/net/glowstone/net/GlowDatagramServer.java [hunk 2]
```diff
@@ -18,6 +18,12 @@
     protected final EventLoopGroup group;
     protected final Bootstrap bootstrap;
 
+    /**
+     * Creates an instance for the specified server.
+     *
+     * @param server the associated GlowServer
+     * @param latch TODO: document this parameter
```
Comment (mastercoms) [inline src/main/java/net/glowstone/net/GlowDatagramServer.java]: The countdown latch used during server startup to wait for network server binding.

File: src/main/java/net/glowstone/entity/GlowEntity.java [hunk 3]
```diff
@@ -150,11 +150,11 @@
      */
     protected boolean velocityChanged;
     /**
-     * A counter of how long this entity has existed
+     * A counter of how long this entity has existed.
      */
     protected int ticksLived;
     /**
-     * Vehicle
+     * Vehicle.
```
Comment (mastercoms) [inline src/main/java/net/glowstone/entity/GlowEntity.java]: The entity this entity is currently riding.

File: src/main/java/net/glowstone/entity/GlowEntity.java [hunk 4]
```diff
@@ -107,7 +107,7 @@
      */
     protected final Vector velocity = new Vector();
     /**
-     * Passenger
+     * Passengers.
```
Comment (mastercoms) [inline src/main/java/net/glowstone/entity/GlowEntity.java]: A list of entities currently riding this entity.

File: src/main/java/net/glowstone/entity/GlowEntity.java [hunk 5]
```diff
@@ -390,6 +391,10 @@ public void setVelocity(Vector velocity) {
         velocityChanged = true;
     }
 
+    /**
+     * Returns the change in velocity per physics tick due to gravity.
```
Comment (mastercoms) [inline src/main/java/net/glowstone/entity/GlowEntity.java:395]: Add a newline.

File: src/main/java/net/glowstone/entity/GlowEntity.java [hunk 6]
```diff
@@ -1447,6 +1506,16 @@ public Entity getLeashHolder() throws IllegalStateException {
         return leashHolder;
     }
 
+    /**
+     * <p>Sets the leash on this entity to be held by the supplied entity.</p>
```
Comment (mastercoms) [inline src/main/java/net/glowstone/entity/GlowEntity.java]: Don't use closing `p` tags, and add a new line between new paragraphs.

File: src/main/java/net/glowstone/entity/GlowEntity.java (hunk 5 above)
Comment (Pr0methean) [reply src/main/java/net/glowstone/entity/GlowEntity.java:395]: Done.

**Review by mastercoms — APPROVED** [review]

**Pull request merged**